apps/spruce/src/pages/adminSettings/tabs/EventLogsTab/AdminEventLogTab.test.tsx

```diff
@@ -0,0 +1,159 @@
+import { MockedProvider } from "@apollo/client/testing";
+import { RenderFakeToastContext } from "@evg-ui/lib/context/toast/__mocks__";
+import {
+  renderWithRouterMatch as render,
+  screen,
+  waitFor,
+} from "@evg-ui/lib/test_utils";
+import { ApolloMock } from "@evg-ui/lib/test_utils/types";
+import {
+  AdminEventsQuery,
+  AdminEventsQueryVariables,
+} from "gql/generated/types";
+import { getUserSettingsMock } from "gql/mocks/getSpruceConfig";
+import { ADMIN_EVENT_LOG } from "gql/queries";
+import { EventLogsTab } from "./EventLogsTab";
+
+const wrapper = ({ children }: { children: React.ReactNode }) => (
+  <MockedProvider mocks={[adminEventsMock, getUserSettingsMock]}>
+    {children}
+  </MockedProvider>
+);
+
+describe("admin event log page", async () => {
+  it("shows skeleton loader when loading", async () => {
+    const loadingWrapper = ({ children }: { children: React.ReactNode }) => (
+      <MockedProvider mocks={[]}>{children}</MockedProvider>
+    );
+
+    const { Component } = RenderFakeToastContext(<EventLogsTab />);
+    render(<Component />, { wrapper: loadingWrapper });
+
+    expect(document.querySelector(".leafygreen-ui-jv9p3c")).toBeInTheDocument();
```

Comment:
It's better to pass a `data-cy` to the Skeleton and use that here.
```
return <Skeleton data-cy="admin-events-skeleton" />;
```
```suggestion
    expect(screen.getByDataCy("admin-events-skeleton")).toBeInTheDocument();
```